content/en/opentelemetry/otlp_ingest_in_the_agent.md

```diff
@@ -193,7 +193,7 @@ There are many other environment variables and settings supported in the Datadog
         fieldRef:
           fieldPath: status.hostIP
     - name: OTEL_EXPORTER_OTLP_ENDPOINT
-      value: "http://$(HOST_IP):4317" # sends to gRPC receiver on port 4317
+      value: "grpc://$(HOST_IP):4317" # sends to gRPC receiver on port 4317
```

Comment:
Our documentation is correct. gRPC is HTTP. The URL doesn't need to change. Why do you think this change is necessary? Please take a look at https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options. There is an example there.